hvpy/api_groups/jpeg2000/tests/test_get_jp2_image.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+from hvpy.api_groups.jpeg2000.get_jp2_image import getJP2ImageInputParameters
+from hvpy.core import execute_api_call
+
+
+def test_str_response():
+    date = datetime(2022, 1, 1, 23, 59, 59)
+    input = {"date": date, "sourceId": 14, "jpip": True, "Json": False}
+    input = getJP2ImageInputParameters(**input)
```

Comment:
Ideally, I would like to avoid setting variables that are keywords used by Python.